paint/data/stac_client.py

```diff
@@ -0,0 +1,560 @@
+import logging
+import pathlib
+from datetime import datetime
+from typing import Optional, Union
+
+import pystac
+import requests
+
+import paint.util.paint_mappings as mappings
+from paint.util import set_logger_config
+
+set_logger_config()
```

Comment:
```suggestion
```
Setting the logger config is only necessary in the very top-level main script to be executed (Juan once told me this 🙂).